internal/webhook/external/github.go

```diff
@@ -59,10 +59,10 @@ func githubHandler(
 		// TODO(fuskovic): eventually switch on event type to perform
 		// different actions (e.g. refresh Promotion on PR merge)
 		eventType := r.Header.Get("X-GitHub-Event")
-		if eventType != "push" {
+		if eventType != "push" && eventType != "ping" {
```

Comment:
This conditional will start to look unwieldy as we handle more events (like PRs being merged) and the same goes for the error message a few lines down. I would suggest that a switch here will be cleaner over time and that the error message can just say the event isn't supported instead of listing all those that are, so that message won't need to be updated for each new event.

```
switch(eventType) {
case: "ping", "push":
default:
	xhttp.WriteErrorJSON(
		w,
		xhttp.Error(fmt.Errorf("event type %q is not supported", eventType),
		http.StatusNotImplemented,
	)
	return
}
```